pkg/southbound/synchronizer/synchronizer.go

```diff
@@ -86,14 +90,14 @@ func New(context context.Context, changeStore *store.ChangeStore, configStore *s
 
```

Comment:
At 2 lines up - should this be **if config == nil** and the **err != nil** needs to be handled